dotnet/src/Microsoft.Agents.AI.Workflows/Specialized/AIAgentHostExecutor.cs

```diff
@@ -54,17 +54,10 @@ protected override async ValueTask TakeTurnAsync(List<ChatMessage> messages, IWo
         emitEvents ??= this._emitEvents;
         IAsyncEnumerable<AgentRunResponseUpdate> agentStream = this._agent.RunStreamingAsync(messages, this.EnsureThread(context), cancellationToken: cancellationToken);
```

Comment:
If emitEvents is false, should this just be using RunAsync, or is there a reason it still needs to use RunStreamingAsync even if it doesn't need to output individual events for each update?